src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -800,4 +824,30 @@ TEST_F(LiveFunctionsDataViewTest, ColumnSortingShowsRightResults) {
     // Sort by descending
     { sort_and_verify(column, orbit_data_views::DataView::SortingOrder::kDescending); }
   }
+}
+
+TEST_F(LiveFunctionsDataViewTest, OnDataChangeResetsHistogram) {
+  EXPECT_CALL(app_, ShowHistogram(nullptr, "", orbit_grpc_protos::kInvalidFunctionId)).Times(1);
+
+  view_.OnDataChanged();
+}
```

Comment:
Sorry for not proposing this while reviewing the previous PRs, I would suggest to change 
```cpp
void LiveFunctionsDataView::OnDataChanged() {
  app_->ShowHistogram(nullptr, "", orbit_grpc_protos::kInvalidFunctionId);
  functions_.clear();
  indices_.clear();
  timer_durations_.clear();
  ...
}
```
to call `UpdateHistogramWithFunctionIds` instead of `ShowHistogram`.

Namely,
```cpp
void LiveFunctionsDataView::OnDataChanged() {
  UpdateHistogramWithFunctionIds({});
  functions_.clear();
  indices_.clear();
  timer_durations_.clear();
  ...
}

```

I feel like "update" something on data changed make more sense here.